modules/fedimint-mint-client/src/lib.rs

```diff
@@ -227,12 +228,21 @@ impl Decodable for OOBNotes {
     }
 }
 
+const BASE64_URL_SAFE: base64::engine::GeneralPurpose = base64::engine::GeneralPurpose::new(
+    &base64::alphabet::URL_SAFE,
+    base64::engine::general_purpose::PAD,
+);
+
 impl FromStr for OOBNotes {
     type Err = anyhow::Error;
 
     /// Decode a set of out-of-band e-cash notes from a base64 string.
     fn from_str(s: &str) -> Result<Self, Self::Err> {
-        let bytes = base64::decode(s)?;
+        let bytes = if let Ok(bytes) = BASE64_URL_SAFE.decode(s) {
+            bytes
+        } else {
+            base64::engine::general_purpose::STANDARD.decode(s)?
+        };
```

Comment:
Looked at the standard and there seems to be no case where there is a valid non-URL-safe base64 string that is also a valid url-safe base64 string encoding different data.